lib/cli.bash

```diff
@@ -18,7 +18,43 @@ function _omb_cmd_reload {
   echo 'Not yet implemented'
 }
 function _omb_cmd_theme {
-  echo 'Not yet implemented'
+  case "$1" in
+  list)
+      local -a available_themes
+      _comp_cmd_omb__get_available_themes
+      for theme in "${available_themes[@]}"; do
+        echo "$theme"
+      done
+    ;;
+  use)
+    local theme=$2
+    if [[ -z "$theme" ]]; then
+      echo 'Usage: omb theme use <theme>'
+      return 2
+    fi
+    local -a available_themes
+    _comp_cmd_omb__get_available_themes
+    for i in "${available_themes[@]}"; do
+      if [ "$i" == "$theme" ]; then
+        _omb_module_require_theme "$theme"
```

Comment:
`_omb_module_require_theme` is kind of "include once"; it checks whether the module is already loaded and loads the module only when it hasn't been loaded. So, with this implementation, one cannot switch to the theme that was loaded before.

To handle this, we may need to factorize a part of `_omb_module_require` (`oh-my-bash.sh`) into `_omb_module_search` and use it to obtain the location of the theme file.